deploy/docker-compose.nonroot.yml

```diff
@@ -8,15 +8,29 @@
 # !! If you use a .env file, use single quotes instead of escaping !!
 
 version: "3"
+
+volumes:
+  timetagger:
+    driver: local
+    
 services:
   timetagger:
-    image: ghcr.io/almarklein/timetagger:v23.9.2-nonroot
+    image: ghcr.io/almarklein/timetagger:v23.11.4-nonroot
     ports:
-      - "80:80"
+      - "8888:80"
     volumes:
-      - ./_timetagger:/opt/_timetagger
+      - timetagger:/root/_timetagger
```

Comment:
This is the compose file for nonroot, so it should not be root 😉

Reply:
Oh my bad, how I forgot that!